platform/testing/contract-testing/how-to-use-pact-for-contract-testing.md

```diff
@@ -608,9 +621,9 @@ https://vagov-pact-broker.herokuapp.com/pacts/provider/VA.gov%20API/consumer/HCA
 
 After the vets-api feature branch is merged to master, that pact should be able to pass verification.
 
-The CircleCI build for vets-api is still under construction with respect to parallelizing our test suite. Tests currently fail simplecov minimum coverage standards due to processes overriding results from other parallel test processes. A solution is in the works to properly merge simplecov results. 
+The CircleCI build for vets-api is still under construction with respect to parallelizing our test suite. Tests currently fail simplecov minimum coverage standards due to processes overriding results from other parallel test processes. A solution is in the works to properly merge simplecov results.
 
-The pact verification functionality is in working order for developer usage when a commit is pushed or when invoked via webhook from the pact broker during a contract content changed event. 
+The pact verification functionality is in working order for developer usage when a commit is pushed or when invoked via webhook from the pact broker during a contract content changed event.
```

Comment:
maybe add something here like: The webhook/pact verification results can be viewed in Circle.

Reply:
I also added functionality to the pact_verification workflow when triggered via webhook to create an artifact containing consumer tags and other information

Reply:
Nice improvement! I've updated this section on both counts.